csharp-basics/exercises/Collections/Exercise5/Program.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Exercise5
+{
+    class Program
+    {
+        static int SquareDigits(int passedNumber)
+        {
+            int squared = 0;
+
+            List<int> squaredNumbers = new List<int>();
+            int length = passedNumber.ToString().ToCharArray().ToList().Count();
```

Comment:
kapēc tik sarežģīti?
```suggestion
            int length = passedNumber.ToString().Length;
```